Docs/source/usage/parameters.rst

```diff
@@ -1315,6 +1315,12 @@ Numerics and algorithms
     ``amr.max_level = 1``. More information can be found at
     https://ieeexplore.ieee.org/document/8659392.
 
+* ``warpx.do_multi_J`` (`0` or `1`; default: `0`)
+    Whether to use the multi-J algorithm, where current deposition and field update are performed multiple times within each time step. The number of sub-steps is determined by the input parameter ``warpx.do_multi_J_n_depositions``. Unlike sub-cycling, field gathering is performed only once per time step, as in regular PIC cycles.
+
+* ``warpx.do_multi_J_n_depositions`` (integer; default: `1`)
```

Comment:
I turn this question to @RemiLehe and @oshapoval, who know better what typical values seemed to work in Warp generally.

Reply:
We discussed this offline, and if `J_linear_in_time` is used then the default of `1`, even if it does not subcycling, already adds some improvement since we change the centering of the `j` to the same time as the `rho` in that case. Generally speaking, `2` seems to be the robust value from your tests.

Reply:
OK, thanks. Does it mean that we want to keep `do_multi_J_n_depositions = 1` as default, then?

Reply:
To do, discussed offline: no default (but parse only when `warpx.do_multi_J` is used).

Reply:
Done in https://github.com/ECP-WarpX/WarpX/pull/1828/commits/c51b7fba63451a31bcc6d474cf8e6f05bde6ec93.